src/svgutils/SVGImporter.as

```diff
@@ -271,7 +271,7 @@ public class SVGImporter {
 			imagesLoaded++;
 			if ((imagesLoaded == imageCount) && (whenDone != null)) whenDone(root);
 		}
-		var imageCount:int, imagesLoaded:int;
+		var imageCount:int = 0, imagesLoaded:int = 0;
```

Comment:
This was causing a warning in IDEA, which I encountered when I was verifying that all paths would eventually call `whenDone`.